python/lsst/ap/association/dia_object.py

```diff
@@ -22,45 +22,47 @@
 
 from __future__ import absolute_import, division, print_function
 
+import numpy as np
+
 import lsst.afw.table as afwTable
+import lsst.afw.geom as afwGeom
+from lsst.afw.coord import averageCoord
+
+__all__ = ["DIAObject", "make_minimal_dia_object_schema"]
 
 
-def make_mimimal_dia_object_schema():
+def make_minimal_dia_object_schema():
     """ Define and create the minimal schema required for a DIAObject.
 
     Return
     ------
     lsst.afw.table.schema.schema.Schema
     """
-    Raise(NotImplementedError)
+
+    schema = afwTable.SourceTable.makeMinimalSchema()
+    # Currently we do not store 
+
+    # For the MVP/S we currently only care about the position though
+    # in the future we will add summary computations for fluxes etc.
+    # as well as their errors.
+
+    # Currently the covariance of a coordinate is not defined in the
+    # stack, so we will hold off until it is implemented. This is to be
+    # addressed in DM-7101.
```

Comment:
Clarified this to say "In the future we would like to use store the convariance but,... etc."